Header header

bool dvl_status
bool imu_status
bool svs_status
